### ISSUE TITLE
Introduction page on tutorial is somewhat redundant with "Getting Started" page.

### ISSUE THREAD

**Issue by soichih** [brainlife/docs#81]
I think this page 
> https://brainlife.io/docs/tutorial/introduction-to-brainlife/

can be merged into the "Getting Started" page here
> https://brainlife.io/docs/user/started/

**Comment by guiomar** [comment]
Yes, I remember having this feeling when I started. I followed both but seemed a bit repetitive, so +1 to merge
